twill-api/src/main/java/org/apache/twill/api/ResourceReport.java

```diff
@@ -61,4 +63,19 @@
    * @return list of services of the application master.
    */
   List<String> getServices();
+
+  /**
+   * Get the map of the log level arguments of the twill application.
+   *
+   * @return the map of the log level arguments of the twill application
+   */
+  Map<String, Map<String, LogEntry.Level>> getLogLevelArguments();
```

Comment:
What's the contract? What's the key? Does it always return all log levels even nothing was set?